eazy/igm.py

```diff
@@ -1,9 +1,243 @@
 import os
 import numpy as np
+#from math import pow as _pow
```

Comment:
I know this was in my original implementation, but could you switch all these to explicit statements like `_pow(1.0+zS, 3.4)` >> `(1.0+zS)**3.4`?

Reply:
The commit 13f75a4 addressed this update.